### ISSUE TITLE
Normalization in `backward` function.

### ISSUE THREAD

**Issue by Jiahao-Ma** [sniklaus/pytorch-pwc#66]
The line of code highlighted in the red bounding box is what I am confused about. The line is for normalization before the interpolation operation. The normalization normally ranges from -1 to 1 in the `torch.nn.functional.grid_sample`. But the normalization seems to range from 0 to 2. 
![image](https://github.com/sniklaus/pytorch-pwc/assets/52849989/b521b88f-45f9-41fe-be9e-d2a25335f075)

**Comment by sniklaus** [comment]
Consider a flow that is -100 pixels along the X axis and an image with a width of 200 pixels. The grid sampler expects values in the range of -1 to 1 and spans a distance of 200 pixels along the X axis. Conversely, we need to map the -100 px from the flow-domain to -1 units in the grid_sampler-domain. This comes out to dividing the flow by half of the image size, and `1/(0.5*x)` is equivalent to `2/x` (what is used in the code).
